### ISSUE TITLE
Specify how "telling stories from the trenches" is implemented in class

### ISSUE THREAD

**Issue by joelostblom** [UBC-MDS/teaching_responsibility#20]
> Moving to the ‘Placing extra emphasis…’ section. I appreciated the authors spelling out specific concerns about version control, especially git and Docker. In the ‘Telling stories from…’ sub-section, I was interested in how the authors actually implement this? Is there a formal ‘story time’? If so, how do the authors manage this, and ensure everyone has a chance to speak? Or is it more informal, and if so, how do the authors ensure this happens each course? Another paragraph or two of detail about actual implementation of this strategy may be useful.

**Comment by joelostblom** [comment]
@ttimbers We do now elaborate on think pair exercises for this in #38, but I have not included a specific example from a class, do you have one that you want to include or do you think the elaborations in the PR of the general practice are sufficient?

**Comment by ttimbers** [comment]
This is how I ran the telling stories from the trenches this year (online teaching):

1. In the next 10 minutes, I want you to think and write down a non-reproducible or non-auditable workflow you have used before at work, on a personal project, or in course work, that negatively impacted your work somehow (make sure to include this in the story). *I then gave my example to help get them started.*
2. When prompted, paste your story in the Google doc (link to be shared in class)
3. Finally, we will take 10 minutes to read at least 3 other stories from the trenches that have been shared.
4. I would highlight and share key themes from the students stories that I could identify when I read them.

So this was really a modification of think pair share... So I didn't have to do the breakout rooms.

In past years in MDS in-person, I would have done:

1.  Prompt students to think about a non-reproducible or non-auditable workflow they have used before at work, on a personal project, or in course work, that negatively impacted your work somehow (make sure to include this in the story). *I then gave my example to help get them started.*
2. With the person beside them, share their story and how it negatively impacted their workflow.
3. Students were then asked to share their story in a course forum or collaborative notetaking document that other students could read (or in smaller classes all groups could share one of their stories aloud with the class).
